src/directory_patcher.rs

```diff
@@ -71,6 +71,14 @@ impl DirectoryPatcher {
             .expect("Error when building file types");
         let mut walk_builder = ignore::WalkBuilder::new(&self.path);
         walk_builder.types(types_matcher);
+        // Note: the walk_builder configures the "ignore" settings of the Walker,
+        // hence the negations
+        if !self.settings.ignored {
+            walk_builder.ignore(true);
+        }
+        if self.settings.hidden {
+            walk_builder.hidden(false);
+        }
```

Comment:
This can just be `walk_builder.hidden(self.settings.hidden)` and such, right?

Reply:
Yeah but since the parameters of walk_builder methods and the fiedls of the settings structs have opposite meanings, I find it clearer this way